javascripts/discourse/components/discourse-tag-banners.js

```diff
@@ -2,15 +2,24 @@ import Component from "@glimmer/component";
 import { tracked } from "@glimmer/tracking";
 import { action } from "@ember/object";
 import { inject as service } from "@ember/service";
+import { getOwner } from "discourse-common/lib/get-owner";
 
 export default class DiscourseTagBanners extends Component {
   @service store;
   @service router;
   @service site;
+  @tracked categoryBannerPresence = null;
   @tracked tag = null;
   @tracked keepDuringLoadingRoute = false;
   @tracked isIntersection = false;
 
+  constructor() {
+    super(...arguments);
+    this.categoryBannerPresence = getOwner(this).lookup(
```

Comment:
We're using this vs the service because when the other component isn't installed, it would fail, right? We should add note here in that case.